### DISCUSSION TITLE
Changing icons

### DISCUSSION THREAD

**Discussion by Zft0** [nroduit/Weasis#300] [Q&A] [answered]
Dear Nicolas,

How is it possible to change icons that Weasis uses for its appearance in the taskbar and in the Start Menu? I've changed contents of logo folder, however, still the initial icons are appeared.

Thanks a lot for your time and attention.

**Comment by nroduit** [comment, marked as answer]
All the resources of the application can be found [here](https://github.com/nroduit/Weasis/tree/master/weasis-distributions/resources). However, I do not recommend to modify the existing resources in order to preserve the consistency in the application.
